label_studio/tasks/models.py

```diff
@@ -684,6 +684,11 @@ class Annotation(AnnotationMixin, models.Model):
         default=None,
         null=True,
     )
+    bulk_created = models.BooleanField(
+        _('bulk created'),
+        default=False,
```

Comment:
This is unsafe, but db_default should be okay!

Reply:
Added `db_default`